sycl/source/event.cpp

```diff
@@ -50,8 +50,18 @@ void event::wait_and_throw(const vector_class<event> &EventList) {
 }
 
 vector_class<event> event::get_wait_list() {
+#ifdef SCHEDULER_20
+  vector_class<event> Result;
+
+  for (std::shared_ptr<cl::sycl::detail::event_impl> &EventImpl :
```

Comment:
Perhaps `auto` is enough, except if you really want to remind that this type is important to know.

Reply:
Done.